flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -208,4 +209,36 @@ public Integer map(Integer value) throws Exception {
 		assertFalse(printConfig.isChainStart());
 		assertTrue(printConfig.isChainEnd());
 	}
+
+//	/**
```

Comment:
According to Apache's rule. Don't comment any code. If you would like to delete them, please delete them. Git logs and pull requests will show the changes.